homeassistant/components/unifi_access/data.py

```diff
@@ -0,0 +1,15 @@
+"""Data structures for UniFi Access integration."""
+
+from dataclasses import dataclass
+
+import uiaccessclient
+
+from . import UniFiAccessDoorCoordinator
+
+
+@dataclass
+class UniFiAccessData:
+    """Data structure for UniFi Access integration."""
+
+    api_client: uiaccessclient.ApiClient
+    door_coordinator: UniFiAccessDoorCoordinator
```

Comment:
Why do we store the api client here as we can just access it via the coordinator?

Reply:
The coordinator carries the specific sub-client for the "space" sub-API, and not the general API client. I was thinking I might need to call multiple sub-APIs in some contexts, for example the the "space" and "device" sub-APIs to build support for devices in the entities. But maybe you guys have a structure where this should always come from the coordinators; and thus coordinated entries should never use clients not also used by their coordinator?